### PR TITLE
ovn multi network plugin kubetron

### PR DESCRIPTION
**Pull request by phoracek**

### CODE REVIEW

**Comment by scollier** [comment]
Excellent.  I'll publish this later this week or early next week.  Thanks for the contribution.

**Comment by phoracek** [comment]
@scollier no problem, thanks :)

File: _posts/2018-05-16-ovn-multi-network-plugin-for-kubernetes-kubetron.markdown [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+Kubetron (working name, `kubernetes + neutron`, quite misleading since we want to support setup without Neutron involved too), allows users to connect their pods to multiple networks configured on OVN. Important part here is, that such networks are configured by an external tool, be it OVN Northbound Database client or higher level tool such as Neutron or oVirt Provider OVN. This allows administrators to configure complicated networks, Kubernetes then only knows enough about the known networks to be able to connect to them - but not all the complexity involved to manage them. Kubetron does not affect default Kubernetes networking at all, default networks will be left intact.
+
+In order to enable use-cases outlined above, KubeVirt will support extended [VirtualMachine Networking API](https://docs.google.com/document/d/10rXr91aqn8MvVcLgHw33WX8BaQwHPZERp25PHxoZGgw/edit?usp=sharing).
```
Comment (fabiand) [inline _posts/2018-05-16-ovn-multi-network-plugin-for-kubernetes-kubetron.markdown]: Here we should have another line explaining the relation to kubevrit a litte more.

"In order to enable the use-cases outlined above, Kubetron can be used to provide multiple interfaces to a pod, further more KubeVirt will then use those interfaces to pass them to it's virtual machines via the in progress VirtualMachine networking API (with link)"

Something along those lines.

Comment (phoracek) [reply _posts/2018-05-16-ovn-multi-network-plugin-for-kubernetes-kubetron.markdown]: Thanks, should be fixed.

**Comment by phoracek** [comment]
@fabiand thanks, should be ok now.

**Pull request merged**